capstone/src/main/java/com/step902020/capstone/Organization.java

```diff
@@ -0,0 +1,83 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+import java.util.*;
+
+@Entity(name = "organization")
+public class Organization {
+  @Id
+  Long datastoreId;
+
+  Long timestamp;
+
+  String name;
+
+  String email;
+
+  String university;
+
+  @Field(name="user-type")
+  String userType;
+  String description;
+  String image;
+  
+  public Organization() {
+
+  }
+  
+  public Organization(Long timestamp, String name, String email, String university, String userType, String description, String image) {
+    this.timestamp = timestamp;
```

Comment:
also just wondering, is there a reason to collect timestamps?

Reply:
Its not really relevant to us right now, but timestamps usually help to visualize growth (like how many users registered and started using the app in a month). I figured it would be a pain to add it later if we ever need it.